pkgs/by-name/cn/cnsprcy/package.nix

```diff
@@ -1,33 +1,51 @@
 {
-  lib,
+  lib, # Dependencies/arguments
   fetchFromSourcehut,
   rustPlatform,
   sqlite,
 }:
 
-rustPlatform.buildRustPackage {
+rustPlatform.buildRustPackage rec {
+  # Main package definition
   pname = "cnsprcy";
-  version = "0.2.0";
+  version = "0.3.0";
 
   src = fetchFromSourcehut {
+    # Fetching the sourcecode
     owner = "~xaos";
     repo = "cnsprcy";
-    rev = "v0.2.0";
-    hash = "sha256-f+DauSU4bT3EljY8/ig7jLnUgyDPEo2NSBQcPN0iKx0=";
+    rev = "v0.3.0";
+    hash = "sha256-OBNl2ZZi1jmJjJvzCwaIN9DSYjGaEo0g4o7/vVOF5aQ=";
   };
 
-  useFetchCargoVendor = true;
-  cargoHash = "sha256-lPTufjKOXMvPy+cP1UyVCTfRXkOmzZqDR6yraIgk+Dg=";
+  cargoLock = {
```

Comment:
We can add a comment here that upstream no longer includes the lockfile, which could be an issue we can raise with them later on.

Reply:
Thanks, @supinie!